packages/flutter/lib/src/widgets/localizations.dart

```diff
@@ -288,7 +288,7 @@ class Localizations extends StatefulWidget {
     assert(context != null);
     assert(type != null);
     final _LocalizationsScope scope = context.inheritFromWidgetOfExactType(_LocalizationsScope);
-    return scope.localizationsState.resourcesFor<T>(type);
+    return scope?.localizationsState?.resourcesFor<T>(type);
```

Comment:
Please document that this returns null if there's no _LocalizationsScope in scope.